### PR TITLE
Add Chinese tasks (C-MTEB)

### PR DESCRIPTION
**Pull request by staoxiao**

### CODE REVIEW

**Review by Muennighoff — APPROVED** [review]
Looks great, amazing job!

Sent you a few more comments via mail~

File: mteb/evaluation/evaluators/RerankingEvaluator.py [hunk 1]
```diff
@@ -71,17 +71,22 @@ def compute_metrics_batched(self, model):
         all_mrr_scores = []
         all_ap_scores = []
 
+        # using encode_queries and encode_corpus functions if they exists,
+        # which can be defined by users to add different instructions for query and passage conveniently
+        encode_queries_func = model.encode_queries if hasattr(model, 'encode_queries') else model.encode
+        encode_corpus_func = model.encode_corpus if hasattr(model, 'encode_corpus') else model.encode
```
Comment (NouamaneTazi) [inline mteb/evaluation/evaluators/RerankingEvaluator.py:77]: Nice idea! I think we should add this liberty to all the difference evaluations which would like to have different encodings for different samples.

Can we please do the same change for the other method below: `compute_metrics_individual()`

File: mteb/tasks/Retrieval/CMTEBRetrieval.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            'eval_splits': ['dev'],
+            'eval_langs': ['zh'],
+            'main_score': 'ndcg_at_10',
+        }
+
+    def load_data(self, **kwargs):
+        if self.data_loaded:
+            return
+
+        self.corpus, self.queries, self.relevant_docs = load_retrieval_data(self.description['hf_hub_name'], self.description['eval_splits'])
```
Comment (NouamaneTazi) [inline mteb/tasks/Retrieval/CMTEBRetrieval.py:212]:
```suggestion
        self.corpus, self.queries, self.relevant_docs = load_retrieval_data(self.description['hf_hub_name'],
                                                                            self.description['eval_splits'])
```

File: README.md [hunk 3]
```diff
@@ -181,7 +181,9 @@ evaluation.run(model)
 ## Leaderboard
 
 The MTEB Leaderboard is available [here](https://huggingface.co/spaces/mteb/leaderboard). To submit:
-1. Run on MTEB: You can reference [scripts/run_mteb_english.py](https://github.com/embeddings-benchmark/mteb/blob/main/scripts/run_mteb_english.py) for all MTEB English datasets used in the main ranking. Advanced scripts with different models are available in the [mteb/mtebscripts repo](https://github.com/embeddings-benchmark/mtebscripts).
+1. Run on MTEB: You can reference [scripts/run_mteb_english.py](https://github.com/embeddings-benchmark/mteb/blob/main/scripts/run_mteb_english.py) for all MTEB English datasets used in the main ranking. 
+Reference [scripts/run_mteb_chinese.py](https://github.com/embeddings-benchmark/mteb/blob/main/scripts/run_mteb_chinese.py) for all MTEB Chinese datasets used in the main ranking. 
+Advanced scripts with different models are available in the [mteb/mtebscripts repo](https://github.com/embeddings-benchmark/mtebscripts).
```
Comment (NouamaneTazi) [inline README.md:186]:
```suggestion
1. Run on MTEB: You can reference [scripts/run_mteb_english.py](https://github.com/embeddings-benchmark/mteb/blob/main/scripts/run_mteb_english.py) for all MTEB English datasets used in the main ranking, or [scripts/run_mteb_chinese.py](https://github.com/embeddings-benchmark/mteb/blob/main/scripts/run_mteb_chinese.py) for the Chinese ones. 
Advanced scripts with different models are available in the [mteb/mtebscripts repo](https://github.com/embeddings-benchmark/mtebscripts).
```

**Review by NouamaneTazi — CHANGES_REQUESTED** [review]
Very clean PR, thanks a lot! 🙌 
Left some small comments before merging this!

File: mteb/evaluation/evaluators/RerankingEvaluator.py (hunk 1 above)
Comment (staoxiao) [reply mteb/evaluation/evaluators/RerankingEvaluator.py:77]: Very sorry for the late reply. 
I think it's a good idea to change the compute_metrics_individual too.

Comment (Muennighoff) [reply mteb/evaluation/evaluators/RerankingEvaluator.py:77]: Did it already 👍

**Comment by Muennighoff** [comment]
I think everything is resolved - merging this! Feel free to still make changes later 😇

**Review by NouamaneTazi — APPROVED** [review]
LGTM! Thanks 🚀

**Pull request merged**